### ISSUE TITLE
missing electromagnet recipe

### ISSUE THREAD

**Issue by tommmmmmmmm** [GTNewHorizons/GT-New-Horizons-Modpack#17842]
### Your Pack Version

720

### Your Server

SP

### Java Version

Java 21

### Type of Server

None

### Your Expectation

![image](https://github.com/user-attachments/assets/05d75c22-94c4-4571-92f1-3c3480b2995a)
electromagnet only "crafting recipe" is from lootbag

i know no one uses that because we just jump on the dislocator, but the the better one requires that electromagnet (which everyone probably has laying around in some chest anyways from lootbags, but it wouldn't be bad if it had a recipe?)
![image](https://github.com/user-attachments/assets/e8da7c03-ced5-40e9-a63a-a76e8ef905d7)

### The Reality

.

### Your Proposal

.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
![image](https://github.com/user-attachments/assets/8bd05e38-12fc-4029-b203-f022d28632cf)

looks fine to me beta3 though with latest gt

**Comment by AbdielKavash** [comment]
@tommmmmmmmm, make sure you are looking for a recipe for the unpowered magnet, not the fully charged one. Comparing yours and chochem's screenshots, the unpowered can only be crafted, and the fully charged can only be obtained as loot. (Or, obviously, by charging the unpowered one.)

**Comment by tommmmmmmmm** [comment]
i was looking on the recipe for the better dislocator, then clicked on the recipe of the magnet i was missing and NEI only showed my picture.

if look for a magnet on NEI, i can find the crafting table recipe but i do not see it as being a lootbag reward.

can't we get those recipes to somehow merge like every other battery?

but i think that's something barely used, so it's not that important

**Comment by chochem** [comment]
so the recipe is fine actually?

**Comment by tommmmmmmmm** [comment]
yes, it's just NEI treating the 2 magnets as totally different items
